templates/sleep/generic-depthcharge-tftp-ramdisk-sleep-template.jinja2

```diff
@@ -0,0 +1,27 @@
+{% extends 'base/kernel-ci-base-tftp-deploy.jinja2' %}
```

Comment:
Maybe extend `generic-depthcharge-tftp-ramdisk-template.jinja2` instead and remove the boot commands below?

Reply:
Yep, that worked fine, thanks!